api/v1beta1/registryconfig_types.go

```diff
@@ -125,15 +126,27 @@ const (
 	PendingState State = "Pending"
 )
 
+type ConditionType string
+
+const (
+	ConditionTypeRegistryCacheValidated  ConditionType = "RegistryCacheValidated"
+	ConditionTypeRegistryCacheConfigured ConditionType = "RegistryCacheConfigured"
+)
+
 type ConditionReason string
 
 const (
-	ConditionReasonRegistryCacheConfigured     ConditionReason = "RegistryCacheConfigured"
-	ConditionReasonFailedToGetSecret           ConditionReason = "FailedToGetCredentialsSecret"
-	ConditionReasonSecretHasIncorrectStructure ConditionReason = "SecretHasIncorrectStructure"
-	ConditionReasonFailedToResolveRegistryURL  ConditionReason = "FailedToResolveRegistryURL"
+	ConditionReasonRegistryCacheValidated        ConditionReason = "RegistryCacheValidated"
+	ConditionReasonRegistryCacheValidationFailed ConditionReason = "RegistryCacheValidationFailed"
+
+	ConditionReasonRegistryCacheConfigured                        ConditionReason = "RegistryCacheConfigured"
+	ConditionReasonRegistryCacheExtensionConfigurationFailed      ConditionReason = "ConditionReasonRegistryCacheExtensionConfigurationFailed"
```

Comment:
We should remove `ConditionReason` prefix.

Reply:
Done